tests/src/test/scala/system/basic/WskRestUnicodeSwift3Tests.scala

```diff
@@ -1,34 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package system.basic
-
-import org.junit.runner.RunWith
-import org.scalatest.junit.JUnitRunner
-
-import common.JsHelpers
-import common.WskTestHelpers
-import common.rest.WskRest
-
-@RunWith(classOf[JUnitRunner])
-class WskRestUnicodeSwift3Tests extends WskUnicodeTests with WskTestHelpers with JsHelpers {
-
-  override val wsk: common.rest.WskRest = new WskRest
-  override lazy val actionKind = "swift:3"
-  override lazy val actionSource = "unicode.swift"
```

Comment:
Hmm, do we still have a unicode test for Swift?

Reply:
yes, this is not related to webactions